app/src/main/java/fr/free/nrw/commons/di/NetworkingModule.java

```diff
@@ -34,7 +34,9 @@
     private static final String WIKIDATA_SPARQL_QUERY_URL = "https://query.wikidata.org/sparql";
     private final String WIKIMEDIA_CAMPAIGNS_BASE_URL =
             "https://raw.githubusercontent.com/commons-app/campaigns/master/campaigns.json";
-    private static final String TOOLS_FORGE_URL = "https://tools.wmflabs.org/";
+    private static final String TOOLS_FORGE_URL = "https://tools.wmflabs.org/urbanecmbot/commonsmisc";
+
+    private static final String TEST_TOOLS_FORGE_URL = "https://tools.wmflabs.org/commons-android-app/tool-commons-android-app";
```

Comment:
Need to change the test URL before merging.

Reply:
Can this be changed now?

Reply:
Had already changed it in my second commit. This is no longer getting used anywhere. Have kept it so that it can be used for testing in the future.